README.md

```diff
@@ -95,7 +95,7 @@ performance results and a comparison with the `async` package.
 
 ## Installing and using
 
-Please see [INSTALL.md](INSTALL.md) for instructions on how to use streamly
+Please see [INSTALL.md](./INSTALL.md) for instructions on how to use streamly
```

Comment:
I am hoping that this tricks Hackage to link properly. The source code indicates that it just might work, we'll know once we upload the rc.